.github/workflows/plugins.yml

```
@@ -17,30 +18,36 @@ on:
 
 jobs:
   sanity:
-    name: Sanity (${{ matrix.ansible }})
+    name: Sanity (${{ matrix.ansible }}) py${{ matrix.python }}
     strategy:
       matrix:
         ansible:
-        - stable-2.10
-        - devel
+          # - stable-2.9 # Only if your collection supports Ansible 2.9
+          - stable-2.10
+          - devel
+        python:
+          - 2.7
+          - 3.7
+          #- 3.8 blocked by https://github.com/ansible/ansible/pull/70155
```

I think I'd prefer this style, but up to you:
```suggestion
          - 3.8
          exclude:
          - python: 3.8  # blocked by https://github.com/ansible/ansible/pull/70155
```

---

Ah, this is much more readable. thans